HotelBooking/HotelBooking.BusinessLogic/Services/Implementation/ReservationService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using HotelBooking.BusinessLogic.Dtos;
+using HotelBooking.BusinessLogic.Services.Abstraction;
+using HotelBooking.DataAccessLayer.Repositories.Interfaces;
+
+namespace HotelBooking.BusinessLogic.Services.Implementation;
+public class ReservationService : IReservationService
+{
+    private IReservationRepository _iReservationRepository;
```

Comment:
To samo co wyżej z nazwą. nazwijmy _reservationRepository i dajmy to pole jako readonly